common/StatBlock.ts

```diff
@@ -16,6 +16,11 @@ export interface NameAndModifier {
   Modifier: number;
 }
 
+export interface NameAndComputedModifier {
+  Name: string;
+  ModifierFormula: string;
```

Comment:
This rename and type change is problematic- I'd like to avoid data migrations on any of the interfaces in `/common/` without an extremely compelling reason. Additive changes are ok; maybe consider instead adding a `ModifierFormula?: string;` optional member to the `NameAndModifier` interface.

Reply:
Yeah definitely. I'll work on it shortly.

As-is it was actually failing the StatBlockImporter tests anyway...